kaspresso/src/main/kotlin/com/kaspersky/kaspresso/instrumental/InstrumentalDepsAssistant.kt

```diff
@@ -0,0 +1,16 @@
+package com.kaspersky.kaspresso.instrumental
+
+import android.app.UiAutomation
+import android.os.Build
+import androidx.annotation.RequiresApi
+import androidx.test.uiautomator.UiDevice
+
+interface InstrumentalDepsAssistant {
+
+    val isInstrumentalEnvironment: Boolean
+    val uiDevice: UiDevice
+    val uiAutomation: UiAutomation
+
+    @RequiresApi(Build.VERSION_CODES.N)
```

Comment:
I believe UiAutomation is available form JELLY_BEAN, API level 16.
Is there any other reason for this?